README.md

```diff
@@ -15,28 +46,44 @@ npm i @po.et/poet-js
 
 ## Usage
 
-The main function you'll be using is `createClaim`:
+First create a `ClaimSigner` to sign and verify your claims. Note that the Po.et network currently uses 
+[Ed25519Signature2018](https://w3c-dvcg.github.io/lds-ed25519-2018/), which requires a Base58
+form of the Ed25519 Private Key. You can use the KeyHelper utility to generate a base58 public/privateKey pair, if you
+do not yet have one.
+
+```typescript
+import { ClaimSigner, KeyHelper } from '@po.et/poet-js'
+
+const { privateKey } = KeyHelper.generateED25519Base58Keys('password') // e.g 'LWgo1jraJrCB2QT64UVgRemepsNopBF3eJaYMPYVTxpEoFx7sSzCb1QysHeJkH2fnGFgHirgVR35Hz5A1PpXuH6'
```

Comment:
👍 on KeyHelper, this is good improvement

Reply:
We should expose it as a CLI call.

Reply:
Would love to be able to `npm i -g @po.et/poet-js` and then `poet <some command` from the shell if this is what you mean. I can help set that up if needed.

Reply:
Is `password` a passphrase that encrypts the private key? Is it entropy?

Reply:
Should we add a small disclaimer about us not guaranteeing the provided examples necessarily be safe to copy and paste, and that implementers should apply their common sense and cryptographic knowledge?

Reply:
Yes password is an entropy element. It does not encrypt the privateKey. I'll rename it.